pkg/master/controller.go

```diff
@@ -257,18 +260,20 @@ type EndpointReconciler interface {
 // masterCountEndpointReconciler reconciles endpoints based on a specified expected number of
 // masters. masterCountEndpointReconciler implements EndpointReconciler.
 type masterCountEndpointReconciler struct {
-	masterCount      int
-	endpointRegistry endpoint.Registry
+	masterCount       int
+	endpointRegistry  endpoint.Registry
+	configmapRegistry configmap.Registry
```

Comment:
Related question: Should config updates then be made via the file or updated directly in etcd? The latter feels more like the "dynamic config" as discussed for the Kubelet, where the file is a fallback if the information is absent from etcd.